pom.xml

```diff
@@ -75,25 +72,21 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-jetty</artifactId>
-            <version>${springboot.version}</version>
```

Comment:
Are you happy taking all these versions out? I'm fine with it, but generally we do tend to like to pin things